frontend/src/pages/about/PageOutreach.vue

```diff
@@ -6,60 +6,108 @@
   <AppSection>
     <AppHeading>Outreach</AppHeading>
     <p>
-      The Monarch Initiative is a community effort, and we welcome your participation!
+      The Monarch Initiative is a community effort, and we welcome your
+      participation!
     </p>
     <AppSection>
       <AppHeading>Ways to Get Involved</AppHeading>
 
       <ul>
         <li>
-          <a href="https://oboacademy.github.io/obook/courses/monarch-obo-training/" target="_blank">OBO Academy: Monarch Training Series</a>
+          <a
```

Comment:
You should use the AppLink component here. This allows all links across the site to have consistent behavior, such as having a little arrow icon for external links like this.

Might also take this opportunity to check the other about pages for `<a>`'s that I may have missed.